helm-deploy-test/tasks/cf-deploy.sh

```diff
@@ -20,12 +20,20 @@ UAA_PORT=2793
 
 CF_NAMESPACE=scf
 UAA_NAMESPACE=uaa
+CAP_DIRECTORY=s3.scf-config
 set +o allexport
 
-unzip s3.scf-config/scf-*.zip -d s3.scf-config/
+# For upgrades tests
+if [ -n "${CAP_INSTALL_VERSION:-}" ]; then
```

Comment:
`:-` isn't necessary in this expansion

Reply:
I just used the format we are already using https://github.com/SUSE/cf-ci/blob/master/helm-deploy-test/tasks/cf-deploy.sh#L34-L43
I guess it should be another PR to address all such changes

Reply:
pretty sure I made the same comment back then. You should avoid putting things into code that serve no function, unless they're making the code more readable. Consider the following two snippets:

```
if [ -n ${SOME_FLAG} ]; then
  echo "SOMEFLAG was set"
fi
```

vs.


```
if [ -n ${SOME_FLAG} ]; then
  if true;
    echo "SOMEFLAG was set"
  fi
fi
```

Since the two snippets are functionally equivalent, but the latter one contains unnecessary characters which make it less straightforward, you should prefer the first form to the latter.


In this case, testing that ${VAR:-} is functionally equivalent to testing that ${VAR} is set, but this is less apparent then the example above (due to the different types of variable expansions), so it causes even more unnecessary confusion.